src/main/java/com/kuri0/rawinput/RawInput.java

```diff
@@ -49,15 +49,26 @@ public void run() {
 						try {
 							Controller[] controllers;
 							controllers = createDefaultEnvironment().getControllers();
-								for (int i = 0; i < controllers.length; i++) {
-			        			if (controllers[i].getType() == Controller.Type.MOUSE) {
-			        				controllers[i].poll();
-			        				if (((Mouse)controllers[i]).getX().getPollData() != 0.0 || ((Mouse)controllers[i]).getY().getPollData() != 0.0) {
-			        					mouse = (Mouse)controllers[i];
-										Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText("Found mouse"));
+							for (Controller controller : controllers) {
+								try {
+									if (controller.getType() == Controller.Type.MOUSE) {
+										controller.poll();
+										float px = ((Mouse) controller).getX().getPollData();
+										float py = ((Mouse) controller).getY().getPollData();
+										float eps = 0.1f;
+
+										// check if mouse is moving
+										if ((-eps < px && px < eps) || (-eps < py && py < eps)) {
```

Comment:
I think this line is causing the problem @xCuri0 is referring to. `((-eps < px && px < eps) || (-eps < py && py < eps))` will return true if either px or py is almost zero. This will cause it to detect a not-moving mouse. Rather, the line should be:
```
if (px < -eps || px > eps || py < -eps || py > eps)
```
That will only trigger the condition if the mouse is actively moving rather than the other way around.